fluent/SignedExtrinsicRune.ts

```diff
@@ -1,10 +1,14 @@
 import { hex } from "../crypto/mod.ts"
+import { $extrinsic } from "../frame_metadata/mod.ts"
 import { Rune, RunicArgs, ValueRune } from "../rune/mod.ts"
 import { Chain, ChainRune } from "./ChainRune.ts"
+import { CodecRune } from "./CodecRune.ts"
 import { ExtrinsicStatusRune } from "./ExtrinsicStatusRune.ts"
 import { PatternRune } from "./PatternRune.ts"
 
 export class SignedExtrinsicRune<out C extends Chain, out U> extends PatternRune<Uint8Array, C, U> {
+  $extrinsic = Rune.fn($extrinsic).call(this.chain.metadata).into(CodecRune)
```

Comment:
Come to think of it... should we move this into `ChainRune`? We could then use `ChainRune`'s `$extrinsic` in `ExtrinsicRune` too.